doc/stdlib/libc.rst

```diff
@@ -40,7 +40,10 @@
 
 .. function:: time(t::TmStruct)
 
-   Converts a ``TmStruct`` struct to a number of seconds since the epoch.
```

Comment:
getting really close here that these are the only losses of data - duplication isn't as much of an issue I think, unless sphinx chokes on things